invenio_vocabularies/records/models.py

```diff
@@ -55,30 +55,31 @@ class VocabularyMetadata(db.Model, RecordMetadataBase):
     __tablename__ = "vocabularies_metadata"
 
 
-class VocabularySubtype(db.Model):
-    """Vocabulary subtype type model.
+class VocabularyScheme(db.Model):
+    """Vocabulary scheme model.
 
-    Current use case is for subject types.
+    This table stores the metadata for schemes (subtypes) of VocabularyType
+    or separate specific vocabularies.
+
+    It is only used to store metadata for subject schemes for now.
+    But we might store affiliations's schemes or other schemes later.
     """
 
-    __tablename__ = "vocabularies_subtypes"
+    __tablename__ = "vocabularies_schemes"
 
     id = db.Column(db.String, primary_key=True)
```

Comment:
I wonder if the `SubjectMetadata` table should have a FK to this.

Reply:
Hmm... I interpreted the codimd to say to place the reference to this id in the json of `SubjectsMetadata` and `id` is human readable e.g. `MeSH` so we don't need joins.